app/models/search_builder/synthetic_category_limit.rb

```diff
@@ -0,0 +1,37 @@
+class SearchBuilder
+  # A search builder module to restrict to a certain SyntheticCollection.
+  # Won't do _anything_ just by being in the chain -- sets class_attributes
+  # on the SearchBuilder that can be used to turn it 'on', either on the class
+  # or even on an instance.
+  #
+  #.    synthetic_category_param : set to a string, and this will be taken
+  #.      from app params to identify a collection to limit to.  If unknown,
+  #.      will do nothing. (Should limit to 0 rows instead?)
```

Comment:
Oh, I think it's possible probably, I'll see if I can do that. Don't expect to run into that case in actual use, but if it's easy to do, then probably makes sense 'just in case' of accidentally doing it.